lib/autoyast.pm

```diff
@@ -744,18 +765,19 @@ sub adjust_user_password {
     return $profile;
 }
 
-=head2 expand_profile_url
+=head2 expand_agama_profile
 
- expand_profile_url($profile, $profile_expanded);
+ expand_agama_profile($profile, $profile_expanded);
 
  Return the PATH of profile with expanded vars
 
 =cut
 
-sub expand_profile_url {
+sub expand_agama_profile {
     my ($profile, $profile_expanded) = @_;
     $profile_expanded //= $profile;
-    my $content = expand_variables(get_test_data($profile));
+    my $content = expand_agama_variables(get_test_data($profile));
+    diag "content=$content";
```

Comment:
this will be updloaded, I guess it is some debug, please remove it.

Reply:
removed.